sql/core/src/main/scala/org/apache/spark/sql/execution/Window.scala

```diff
@@ -198,7 +198,8 @@ case class Window(
           functions,
           ordinal,
           child.output,
-          (expressions, schema) => newMutableProjection(expressions, schema))
+          (expressions, schema) =>
+            newMutableProjection(expressions, schema, subexpressionEliminationEnabled))
```

Comment:
This would definitely help the computation of summary statistics. But would `subexpressionEliminationEnabled` have other side effects? cc @nongli